src/Datepicker/DatesList.tsx

```diff
@@ -8,7 +8,6 @@ import { Day } from './types'
 
 type Props = {
   items: Day[]
-  firstDayShift: boolean
```

Comment:
Are we good with removing the prop completely?
From the discussion that we had on Slack, having the calendar-like layout should be the default and I can't see a case where the other one is needed